model/godleyTableWindow.cc

```diff
@@ -86,17 +86,19 @@ namespace minsky
       case 0:
         godleyIcon.table.insertRow(idx+1);
         break;
-      case 1:
-        godleyIcon.deleteRow(idx+1);
+      case 1:      
+        if (pos!=first && pos!=firstAndLast) godleyIcon.deleteRow(idx+1); // Initial conditions row cannot be deleted, even when it is the only row in the table. For ticket 1064
         break;
-      case 2:
-        if (pos==first)
-          godleyIcon.table.moveRow(idx,1);
-        else
-          godleyIcon.table.moveRow(idx,-1);
+      case 2:  
+        if (pos==first && pos==firstAndLast) // Third button cannot move initial conditions row, even when it is the only row in the table. For ticket 1064
+          godleyIcon.table.moveRow(idx,0); 
```

Comment:
Is this a no operation?

Reply:
pos cannot be both first and firstAndLast concurrently, so this stanza is never executed,

Reply:
Thanks, I've removed that operation.

Reply:
But not committed yet?

Reply:
I have decided to fix testCode.cc first. But will commit everything now.